lib/vsc/administration/user.py

```diff
@@ -232,70 +235,92 @@ def pickle_path(self):
         a VscTier2AccountpageUser instance.
         """
         (path, _) = self.institute_path_templates[self.pickle_storage]['user'](self.account.vsc_id)
-        return os.path.join(self.institute_storage[self.pickle_storage].gpfs_mount_point, path)
+        return os.path.join(self.institute_storage[self.pickle_storage].backend_mount_point, path)
 
-    def _create_grouping_fileset(self, filesystem_name, path, fileset_name):
+    def _create_grouping_fileset(self, storage, path, fileset_name):
         """Create a fileset for a group of 100 user accounts
 
         - creates the fileset if it does not already exist
         """
-        self.gpfs.list_filesets()
+        try:
+            filesystem_name = storage.filesystem
+        except AttributeError:
+            logging.exception("Failed to access attribute 'filesystem' in the data storage instance")
+            raise
+
+        try:
+            storage.operator().list_filesets()
+        except AttributeError:
+            logging.exception("Storage backend %s does not support listing filesets", storage.backend)
+            raise
+
         logging.info("Trying to create the grouping fileset %s with link path %s", fileset_name, path)
 
-        if not self.gpfs.get_fileset_info(filesystem_name, fileset_name):
+        if not storage.operator().get_fileset_info(filesystem_name, fileset_name):
             logging.info("Creating new fileset on %s with name %s and path %s", filesystem_name, fileset_name, path)
             base_dir_hierarchy = os.path.dirname(path)
-            self.gpfs.make_dir(base_dir_hierarchy)
-            self.gpfs.make_fileset(path, fileset_name)
+            storage.operator().make_dir(base_dir_hierarchy)
+            storage.operator().make_fileset(path, fileset_name)
         else:
             logging.info("Fileset %s already exists for user group of %s ... not creating again.",
                          fileset_name, self.account.vsc_id)
 
-        self.gpfs.chmod(0o755, path)
+        storage.operator().chmod(0o755, path)
+
+    def _get_storage_partition(self, storage_name):
+        """Seek and return storage partition from institute's storage"""
+        try:
+            storage = self.institute_storage[storage_name]
+        except KeyError:
+            err_msg = "Failed to access storage '%s' in the storage configuration of %s"
+            logging.exception(err_msg, storage_name, self.host_institute)
+            raise
+        else:
+            return storage
 
     def _get_mount_path(self, storage_name, mount_point):
         """Get the mount point for the location we're running"""
         if mount_point == "login":
             mount_path = self.institute_storage[storage_name].login_mount_point
-        elif mount_point == "gpfs":
-            mount_path = self.institute_storage[storage_name].gpfs_mount_point
+        elif mount_point == "backend":
```

Comment:
perhaps better to use a constant for this (and also for `login`)

Reply:
done in https://github.com/hpcugent/vsc-administration/pull/134/commits/1f210803bd6c84daff6606f006e96b164bc7903f